### PR TITLE
[tool] Find merged pr with specific milestone.

### LINKED ISSUE #11289: [Feature][release] New tool find out not cherry-pick pr with specific milestone
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/dolphinscheduler/issues?q=is%3Aissue) and found no similar feature requirement.

### Description

when we release minor version of dolphinscheudler, we will add milestone to issue or PR, and cherry-pick them when PR merged, currently we have to find not cherry-pick PR by manual, maybe we should add a tool to make it easier

### Use case

_No response_

### Related issues

_No response_

### Are you willing to submit a PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by zhongjiajie**
Add script for  easier release including cherry pick,
generating changelog and contributor list

* Auto cherry-pick: `python release.py cherry-pick`
* Generate changelog: `python release.py changelog`
* Generate contributor: `python release.py contributor`

close: https://github.com/apache/dolphinscheduler/issues/11289
related: https://github.com/apache/dolphinscheduler/pull/12222

### CODE REVIEW

**Comment by zhongjiajie** [comment]
* Add more detail about how to use it
* Show command without already cherry pick commit
* (optional) Add auto cherry pick according to the time

File: tools/release/requirements.txt [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+requests~=2.28
+GitPython
```
Comment (zhongjiajie) [inline tools/release/requirements.txt]:
```suggestion
GitPython~=3.1
```

**Comment by zhongjiajie** [comment]
PTAL @caishunfeng @SbloodyS

**Review by SbloodyS — APPROVED** [review]
LGTM

**Pull request merged**